aio/content/examples/getting-started/src/app/shipping/shipping.component.ts

```diff
@@ -11,15 +12,18 @@ import { CartService } from '../cart.service';
   styleUrls: ['./shipping.component.css']
 })
 // #docregion props
-export class ShippingComponent {
+export class ShippingComponent implements OnInit {
 
-  shippingCosts = this.cartService.getShippingPrices();
+  shippingCosts: Observable<{  type: string , price: number}[]>;
 // #enddocregion props
 
 // #docregion inject-cart-service
   constructor(private cartService: CartService) { }
 // #enddocregion inject-cart-service
 
+   ngOnInit(): void {
+    this.shippingCosts =  this.cartService.getShippingPrices();
+   }
 
 // #docregion props
```

Comment:
This should be part of the `props` docregion:

```suggestion
// #docregion props

  ngOnInit(): void {
    this.shippingCosts =  this.cartService.getShippingPrices();
  }

```

Could you also please update the part of the guide where this snippet is used to reflect the change (i.e. mention that the property is initialized in an `ngOnInit()` method)? 🙏 

[1]: https://github.com/angular/angular/blob/42289f25c69ec7a36d2e3ef424d32f77bc7cd19a/aio/content/start/start-data.md?plain=1#L269-L271